### DISCUSSION TITLE
What is the hastag symbol with import?

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#22244] [General]
Can someone please tell me what the hashtag does with an import? I‘ve used it before but never understood what it is. 

```js
import Foo from '#components'
```

**Comment by angelhdzmultimedia** [comment]
Those are **Manual Imports**.

You can also import from `'#imports'` or from `'nuxt/app'`.

For example, yesterday I was playing with `@vue-macros/nuxt` in a `setupSFC`  `.setup.tsx` file, so I had to manually import `import {NuxtPage} from '#components'`.
